Cartfile.resolved

```diff
@@ -1,7 +1,7 @@
-github "Alamofire/Alamofire" "4.7.1"
```

Comment:
Is this supposed to be in this pull request?

Reply:
Nope 🤣  
Now that this is working I will adjust those things :))